tests/test_resource_tracker.py

```diff
@@ -0,0 +1,227 @@
+"""Tests for the SemaphoreTracker class"""
+import errno
+import gc
+import io
+import os
+import pytest
+import re
+import signal
+import sys
+import time
+import warnings
+import weakref
+
+from loky import ProcessPoolExecutor
+import loky.backend.resource_tracker as resource_tracker
+from loky.backend.semlock import sem_unlink
+from loky.backend.context import get_context
+
+
+def _resource_unlink(name, rtype):
+    resource_tracker._CLEANUP_FUNCS[rtype](name)
+
+def get_rtracker_pid():
+    resource_tracker.ensure_running()
+    return resource_tracker._resource_tracker._pid
+
+
+class TestSemaphoreTracker:
+
+    @pytest.mark.skipif(sys.platform == "win32",
+                        reason="no resource_tracker on windows")
```

Comment:
This should not be skipped anymore I guess.

Reply:
(cf. TODO) I was waiting for your input on that.